main.go

```diff
@@ -61,12 +63,15 @@ func main() {
 		Scheme:             scheme,
 		MetricsBindAddress: metricsAddr,
 		LeaderElection:     enableLeaderElection,
+		Namespace:          namespace,
 	})
 	if err != nil {
 		setupLog.Error(err, "unable to start manager")
 		os.Exit(1)
 	}
 
+	ctrl.Log.WithName("Namespace").Info(namespace)
```

Comment:
I think we can say something like `starting manager in namespace <namespace name> ` or just merge this with error message in line 69.

Reply:
+1 to improving error message (if we merge with line 69 it won't show up in success cases).

Reply:
Fair, will update the message. I think it is good to have a log even in case there is no error.

Reply:
done